src/BaseTokenWrapper.sol

```diff
@@ -103,6 +110,24 @@ abstract contract BaseTokenWrapper is Ownable, IBaseTokenWrapper {
     return _withdrawToken(amount, to, aTokenOut);
   }
 
+  /// @inheritdoc IBaseTokenWrapper
+  function borrowToken(
+    uint256 amount,
+    address to,
+    uint16 referralCode
+  ) external virtual {}
+
+  /// @inheritdoc IBaseTokenWrapper
+  function borrowTokenWithPermit(
+    uint256 amount,
+    address to,
+    uint16 referralCode,
+    uint256 deadline,
```

Comment:
can we use `PermitSignature` struct here?

Reply:
Done [1ab43a0](https://github.com/aave/token-wrappers/commit/1ab43a08563f9e8a0bcaa33acf2cefb3d232359f)